flow.yml

```diff
@@ -24,13 +30,15 @@ executors:
       CUDA_VISIBLE_DEVICES: 0  # change this if you have multiple GPU
       XLA_PYTHON_CLIENT_ALLOCATOR: platform  # https://jax.readthedocs.io/en/latest/gpu_memory_allocation.html
     replicas: 1  # change this if you have larger VRAM
-    needs: [gateway]
+    needs: [gateway, clip_encoder]
```

Comment:
what is the logic for this complex graph?

Reply:
`clip_encoder` must run before `diffusion`. Is it the correct way to ensure that?
`gateway` might be redundant here, not sure.

Reply:
gateway is not redundant, is wrong.

Reply:
Plot the `flow` and u will see.